docs/projects/P5/2_projectfinal.md

```diff
@@ -2,17 +2,17 @@
 
 ## Deliverables
 
-**Project Report and Presentation** – due Sunday, May 7th, 11:59pm
+**Project Report and Presentation** – due Monday, December 11th, 11:59pm
 
-- [Project Report](#project-report-120-pts) (120 pts) 
-- [Extra Credit](#project-report-120-pts) (20 pts)
+- [Project Report](#project-report-200-pts) (200 pts) 
+- [Extra Credit](#project-report-200-pts) (20 pts)
 - [Project Presentation Slides](#project-presentations-100-pts) (100 points, graded based on presentation)
 
-**Presentation Date** – Monday, May 8th, 5.00pm - 8:30pm, during exam timeslot
+**Presentation Date** – Tuesday, December 12th, 5:30-8:30pm, during exam timeslot
 
-**Reflection** – due Monday, May 8th, 11:59pm
+**Reflection** – due Tuesday, December 12th, 11:59pm
```

Comment:
Let's make this due on Monday, December 11th as well so that it does not conflict with class presentations.